src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -871,48 +877,95 @@ void makeNotNullMembersMaybeNull()
                     }
                 }
 
+                return;
+
                 ImmutableArray<Symbol> getMembersNeedingDefaultInitialState()
                 {
                     if (_hasInitialState)
                     {
                         return ImmutableArray<Symbol>.Empty;
                     }
 
+                    bool includeRequiredMembers = true;
+                    bool includeBaseRequiredMembers = true;
+                    bool hasThisConstructorInitializer = false;
+
+                    if (method is SourceMemberMethodSymbol { SyntaxNode: ConstructorDeclarationSyntax { Initializer: { RawKind: var initializerKind } } })
+                    {
+                        var baseOrThisInitializer = (_baseOrThisInitializer ?? GetConstructorThisOrBaseSymbol(this.methodMainNode));
```

Comment:
I didn't follow, why do we need two ways of getting the base/this initializer symbol?